packages/genesys-spark-components/src/components/stable/gux-listbox/option-group/gux-option-group.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, h, Host, JSX, Prop, State, Element } from '@stencil/core';
+import { hasSlot } from '@utils/dom/has-slot';
+
+/**
+ * The listbox component provides keyboard bindings and a11y patterns for selecting
+ * from a list of options.
+ *
+ * @slot - collection of elements conforming to the ListboxOptionElement interface
+ */
+@Component({
+  styleUrl: 'gux-option-group.scss',
+  tag: 'gux-option-group',
+  shadow: true
+})
+export class GuxOptionGroup {
+  @Element()
+  root: HTMLElement;
+
```

Comment:
The native `optgroup` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) element has a disabled property we should implement.

Reply:
I'll add in the prop for now. Will have to check with design about styling.